src/tests/ftest/container/cont_rf_oclass_enforcement.yaml

```diff
@@ -0,0 +1,51 @@
+hosts:
+  test_servers:
+    - server-A
+    - server-B
+    - server-C
+    - server-D
+    - server-E
+    - server-F
+  test_clients:
+    - client-G
+timeout: 360
+server_config:
+  name: daos_server
+  servers:
+    targets: 2
+pool:
+  mode: 511
+  name: daos_server
+  scm_size: 1GB
+  svcn: 3
+  control_method: dmg
+  pool_query_timeout: 30
+container:
+  control_method: daos
+  akey_size: 5
+  dkey_size: 5
+  data_size: 5
+  object_qty: 10
+  record_qty: 10
+  cont_property: !mux
+    rf_0:
+      properties: rf:0
+    rf_1:
+      properties: rf:1
+    rf_2:
+      properties: rf:2
+    rf_3:
+      properties: rf:3
+    rf_4:
+      properties: rf:4
+rebuild:
+  traffic_oc: !mux
+    object_class_sx:
+      object_class: OC_SX
+    object_class_rp_2g1:
+      object_class: OC_RP_2G1
+    object_class_rp_3g1:
+      object_class: OC_RP_3G1
```

Comment:
Not sure how i get how this works with RF. For instance, SX isn't compatible with RF1. Could you please elaborate? We should instead use the automatic object class selection https://github.com/daos-stack/daos/blob/master/src/include/daos_obj.h#L404

Reply:
Thanks for the comment.
There are positive and negative in this tests,
ie: 
SX with rf:0 is positive test, and 
     with rf:1 to rf:3 are negative test, expecting DER_ERR -1003 with obj-write.